guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -1,55 +1,271 @@
 
-The *configuration* sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
+Below are listed sub-statuses that {Project} contains.
+ifdef::foreman-el,foreman-deb,katello[]
+There can be more sub-statuses depending on which plug-ins you add to your {Project}.
+endif::[]
 
-To view the sub-status for a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click the host whose full status you want to inspect.
-You can also view substatus information in the hover help for each host.
+Configuration::
+This sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status
+| Alerts disabled | OK
+| Active | OK
+| Pending | OK
+| No changes | OK
+| No reports | OK / Warning
+| Out of sync | Warning
+| Error | Error
+|===
++
+Additional information about the values of this sub-status:
++
+* *Active*: During the last configuration, some resources were applied.
+* *Pending*: During the last configuration, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+* *No changes*: During the last configuration, nothing changed.
+* *No reports*: This can be both a *Warning* or *OK* status.
+When there are no reports but the host uses an associated {SmartProxy} for configuration management or the `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+Otherwise it maps to *OK*.
+* *Error*: This indicates an error during configuration.
+For example, a configuration run failed to install a package.
+* *Out of sync*: A configuration report was not received within the expected interval, based on the `outofsync_interval` setting.
+Reports are identified by an origin and can have different intervals based upon it.
 
-In the *Properties* table of the host details' page, you can view both the global host status and all sub-statuses.
+Build::
+This sub-status is only relevant for hosts provisioned from {Project} or hosts registered through global registration.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Installed | OK | 0
+| Pending installation | OK | 1
+| Token expired | Error | 2
+| Installation error | Error | 3
+|===
 
-Each sub-status can define its own set of possible values that are mapped to the three global status values.
+ifndef::foreman-deb[]
+Compliance::
+Indicates if the host is compliant with OpenSCAP policies.
++
+Possible values:
++
+[options="header"]
+|===
+| Label | Global host status | Number value
+| Compliant | OK | 0
+| Inconclusive | Warning | 1
+| At least one incompliant | Error | 2
+|===
+endif::[]
 
-The *Build* sub-status has two possible values {endash} *pending* and *built* that are both mapped to global *OK* value.
+ifdef::satellite,orcharhino[]
```

Comment:
```suggestion
ifndef::foreman-deb[]
```

Unless OVAL is not connected to "foreman_openscap"? https://docs.theforeman.org/nightly/Managing_Security_Compliance/index-katello.html#Security_Content_Automation_Protocol_security-compliance

Reply:
Well, it is not present on katello build, so I presume it's not gonna be on bare foreman.

Reply:
oval comes from foreman_openscap

Reply:
@maximiliankolb I'd rather omit it since it is not on bare foreman unless you specifically want me to make it visible.